src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -175,6 +175,143 @@ private static IEnumerable<(string testCaseName, string metric, IEnumerable<doub
     }
 
     [Serializable]
+    [XmlRoot("BenchmarkScenario")]
+    public sealed class BenchmarkScenario
+    {
+        [XmlAttribute("Name")]
+        public string Name { get; set; }
+
+        [XmlAttribute("Namespace")]
+        public string Namespace { get; set; }
```

Comment:
It makes sense. Then, I believe we should keep it as it can be used to categorize and group tests.